ingest/Snakefile

```diff
@@ -0,0 +1,87 @@
+from snakemake.utils import min_version
+
+min_version(
+    "7.7.0"
+)  # Snakemake 7.7.0 introduced `retries` directive used in fetch-sequences
+
+if not config:
+
+    configfile: "config/config.yaml"
+
+
+send_slack_notifications = config.get("send_slack_notifications", False)
+
+
+def _get_all_targets(wildcards):
+    # Default targets are the metadata TSV and sequences FASTA files
+    all_targets = ["data/sequences.fasta", "data/metadata.tsv"]
```

Comment:
_nitpick (on my past self)_  

I'm not sure why I set up the monkeypox ingest workflow to output the final results to the `data` directory, maybe I carried it over from the ncov-ingest pipeline. 

I think it'd make more sense for final outputs to be in the `results` directory instead of being mixed with the intermediate data files.

Reply:
Updated upstream in [mpox repo](https://github.com/nextstrain/mpox/pull/222)